Makefile

```diff
@@ -281,10 +281,10 @@ ifort:   # BUILDTARGET Intel Fortran, C, and C++ compiler suite
 	"CC_SERIAL = icc" \
 	"CXX_SERIAL = icpc" \
 	"FFLAGS_PROMOTION = -real-size 64" \
-	"FFLAGS_OPT = -O3 -convert big_endian -free -align array64byte" \
-	"CFLAGS_OPT = -O3" \
-	"CXXFLAGS_OPT = -O3" \
-	"LDFLAGS_OPT = -O3" \
+	"FFLAGS_OPT = -O3 -fp-model precise -convert big_endian -free -align array64byte" \
+	"CFLAGS_OPT = -O3 -fp-model precise" \
+	"CXXFLAGS_OPT = -O3 -fp-model precise" \
+	"LDFLAGS_OPT = -O3 -fp-model precise" \
```

Comment:
I would suggest keeping the compiler flags as they were. I appreciate that adding `-fp-model precise` is useful for development, but this flag does lead to slower execution.

Reply:
done. totally forgot to remove this option after finishing to test the implementation.

Reply:
If you've made changes to the `Makefile` in your local clone, I think you may need to push the changes up to GitHub.